### DISCUSSION TITLE
Unhandled Exception: [unknown] 4:

### DISCUSSION THREAD

**Discussion by wengxianxun** [doppeltilde/firebase_game_services#9] [Q&A] [answered]
E/flutter (22899): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: [unknown] 4: 
E/flutter (22899): #0      MethodChannelFirebaseGameServices.signIn (package:firebase_game_services_platform_interface/method_channel_firebase_game_service_platform.dart:84:7)
E/flutter (22899): <asynchronous suspension>
E/flutter (22899): #1      FirebaseGameServices.signIn (package:firebase_game_services/firebase_game_services.dart:68:12)

**Comment by jonafeucht** [comment]
Sorry for the late reply.
Do you have further debugging info or is that everything? Also, could you post your flutter -v doctor

**Comment by jonafeucht** [comment, marked as answer]
I did a bit of testing, and it's most likely that you either forgot or misconfigured the SHA certificate fingerprints step.

I'd recommend going through the process again and see if you missed something: https://firebase.google.com/docs/auth/android/play-games#set-up-firebase-project.
It's not mentioned anywhere, but you will also need to add your debug sha certificate fingerprints in addition to the release ones.
